django_ipam/static/django-ipam/js/widget.js

```diff
@@ -0,0 +1,18 @@
+/*jslint browser: true*/
+/*global $, jQuery, alert*/
+
+function getAvailableIp() {
+    "use strict";
+    var e = document.getElementById("id_subnet");
+    var subnet = e.options[e.selectedIndex].value;
+    $.ajax({
+        type: 'GET',
+        url: "http://" + window.location.host + "/api/v1/subnet/" + subnet + "/get-first-available-ip",
```

Comment:
never ever hardcode URLs in your frontend code, you need to generate and pass the URL from the python code, otherwise when somebody will change the URLs in the django routing configuration this code will break